### ISSUE TITLE
MySQL's `REGEXP` binary operator is not supported

### ISSUE THREAD

**Issue by accforgithubtest** [lovasoa/SQLpage#107]
I have gotten sqlpage to work with mysql, and able to see the data on UI for a bunch of queries.

However, any query with a REGEXP in it seems to cause the error -
```
Backtrace
sql parser error: Expected an SQL statement, found: REGEXP at Line: 26, Column 15
SQLPage could not parse and prepare this SQL statement
```

Is REGEXP somehow not supported in sqlpage ?
[I have confirmed the exact same query to be working outside sqlpage.]

**Comment by lovasoa** [comment]
SQLPage needs to parse your queries in order to know which parameters to extract from the user's request to pass to the database, and to implement [SQLPage functions](https://sql.ophir.dev/functions.sql).

To do that, it uses a very powerful and versatile SQL parser: [sqlparser-rs](https://github.com/sqlparser-rs/sqlparser-rs). It's the same parser that is used in InfluxDB and Apache DataFusion, among others.

However, it may happen that some syntax that is valid in one database is not supported by the parser. When this happens, you can open an issue here with an example failing query, and we will implement in upstream in sqlparser. In this case, I [opened this issue](https://github.com/sqlparser-rs/sqlparser-rs/issues/1015).

In the mean time, you can use [the traditional function syntax for matching regular expressions](https://dev.mysql.com/doc/refman/8.0/en/regexp.html#function_regexp-instr):

```sql
select regexp_like('hello', '^hell');
```

instead of

```sql
select 'hello' REGEXP '^hell';
```

**Comment by accforgithubtest** [comment]
Thanks for the suggestions and also raising the issue, will give this a spin ! Thank you.

**Comment by lovasoa** [comment]
My patch to sqlparser-rs was merged, we'll soon have support for the `REGEXP` operator in sqlpage :confetti_ball:

**Comment by lovasoa** [comment]
fix released in v0.15

**Comment by accforgithubtest** [comment]
thanks @lovasoa ! Just tried this with latest version and it works !
